src/IndexBuilder.php

```diff
@@ -96,7 +96,7 @@ public function migrate(Index $currentIndex, array $params = [], array $context
         return $newIndex;
     }
 
-    public function purgeOldIndices(string $indexName): array
+    public function purgeOldIndices(string $indexName, bool $dry = false): array
```

Comment:
Can you rename this parameter to `dryRun`, it's more standard

Reply:
I've renamed the parameter to `dryRun`.